accelerator/config.h

```diff
@@ -94,12 +96,14 @@ typedef struct iota_config_s {
 
 /** struct type of accelerator cache */
 typedef struct ta_cache_s {
-  char* host;             /**< Binding address of redis server */
-  uint64_t timeout;       /**< Timeout for keys in redis */
-  char* buffer_list_name; /**< Name of the list to buffer transactions */
-  char* done_list_name;   /**< Name of the list to store successfully broadcast transactions from buffer */
-  uint16_t port;          /**< Binding port of redis server */
-  bool cache_state;       /**< Set it true to turn on cache server */
+  char* host;               /**< Binding address of redis server */
+  uint64_t timeout;         /**< Timeout for keys in redis */
+  char* buffer_list_name;   /**< Name of the list to buffer transactions */
+  char* done_list_name;     /**< Name of the list to store successfully broadcast transactions from buffer */
+  uint16_t port;            /**< Binding port of redis server */
+  bool state;               /**< Set it true to turn on cache server */
+  long int capacity;        /**< The maximum capacity of caching server */
```

Comment:
Unify naming. Here are `redis server`, `redis`, `cache server` and `caching server`.